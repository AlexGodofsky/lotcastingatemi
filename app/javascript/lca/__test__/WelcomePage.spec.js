import React from 'react'
import WelcomePage from '../components/welcomePage/index.jsx'
import renderer from 'react-test-renderer'

test('WelcomePage renders at all', () => {
  const component = renderer.create(
    <WelcomePage />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
