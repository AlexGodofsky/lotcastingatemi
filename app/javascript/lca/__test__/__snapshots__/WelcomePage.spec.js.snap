// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage renders at all 1`] = `
<div
  className="welcomePage"
>
  <h1>
    Lot-Casting Atemi
  </h1>
  <p>
    <strong>
      Cost:
    </strong>
     15m, 1wp, 
    <strong>
      Mins:
    </strong>
     Essence 2
    <br />
    <strong>
      Type:
    </strong>
     Simple
    <br />
    <strong>
      Keywords:
    </strong>
     None
    <br />
    <strong>
      Duration:
    </strong>
     One scene
  </p>
  <p>
    The Solar Exalted have achieved such heights of skill that even their games become effortless.  When a Solar activates Lot-Casting Atemi she instantly becomes aware of her scores and those of the other players. She likewise never forgets the strength of her signature moves, or the number of dice required to play them. She can roll those dice with but a thought, and no matter how distant she is from the game, all participants become aware of her successes.
  </p>
</div>
`;
